docs/introduction/react-for-iac.md

```diff
@@ -0,0 +1,38 @@
+# React for Infrastructure-as-Code
+
+A bold statement? Maybe. 
+
+Solving a real problem like React does? 100%. 
+
+Let's look at the **reasons** behind the claim and then the **similarities** and **differences** between the two frameworks. 
+
+## React for Building UI v. CDF for Building Infrastructure
+
+We developers have long benefited from the arrivals from frameworks that boost productivity when developing using certain languages. Think *Ruby on Rails* to *Ruby*, or *Spring (Boot)* to *Java*. 
+
+The arrival of *React* has completely changed how we develop UI (React DOM on web, React Native on mobile). Yes, we have had *JavaScript* for as many years as Internet, which seems to give us good enough web development experience. But there was nothing like React that gives us a component-based approach while marrying *HTML* and *JavaScript* seamlessly, and seeing *TypeScript* gaining popularity as a result. 
```

Comment:
Should at least acknowledge Angular right?  and qualify to exclude backend web frameworks. At least ASP.NET has WebControls. 
>But there was nothing like React that gives us a component-based approach while marrying *HTML*

Reply:
It's worth adding those specifics then.

To be clear I don't have reason to out right disagree. But because I'm not thinking about all the right properties there are question. Let's assume at least some of our readers will do the same.